### ISSUE TITLE
Define Transmission download folder

### ISSUE THREAD

**Issue by ronilaukkarinen** [barbequesauce/Watcher3#101]
**Is your feature request related to a problem? Please describe.**
When Transmission is getting the file, it starts the download to the directory that is currently relied in Transmission server settings.

**Describe the solution you'd like**
The dl folder should be defined in Watcher, not in Transmission server.

**Comment by barbequesauce** [comment]
This is interesting to me as a transmission user. Can you show an example of this in action? I have Watcher, Sickgear, LazyLibrarian, and Mylar all running today with labels - but the labels form under the transmission data folder...

**Comment by ronilaukkarinen** [comment]
Yeah, this is still happening. I have transmission-daemon running on server and some other apps sometimes change the "default folder download folder on remote host", for example Windows client Transmission Remote GUI changes its back to "Literature" folder for example. Later I see Watcher will put all the films to Literature folder instead of location they should be in. Even "move" setting doesn't help here. Every finished dl I have to manually move to correct dir.

Hmm. How the labels work?

**Comment by barbequesauce** [comment]
Labels append a directory to the base data directory. For example, in my settings.json I have:

`"download-dir": "/mnt/sdb1/completed",`

Then in Watcher I set the label as follows:

<img width="517" alt="Screen Shot 2019-12-29 at 6 47 06 PM" src="https://user-images.githubusercontent.com/25070944/71564270-ee885200-2a6b-11ea-8b07-41dab914ed53.png">

Which results in downloads landing in /mnt/sdb1/completed/Watcher.

Similarly, Mylar is set with a label of "comics"; SickGear with "tv"; LazyLibrarian with "books".

**Comment by ronilaukkarinen** [comment]
My Category setting was left empty as I didn't understand its functionality (I do not use categories, I just dump every file to different hard drives all over the place, heh). However, the problem is here that different apps I use change my transmission download-dir from time to time, it's not permanently the same. So setting a category doesn't fix this.

Watcher should define its own dl dir when snatching a torrent. Sickbeard, CouchPotato, Sonarr and others allow you to do that - Watcher seem not to have that kind of option?

**Comment by barbequesauce** [comment]
I spent some time experimenting with transmission-cli last night - setting the destination directory is clearly available over RPC, so this is feasible. It's next on my to-do list (unless @scambra gets to it first with his transmission & deluge changes?)

**Comment by scambra** [comment]
@barbequesauce you can do that, I'm stuck with deluge issues and not much spare time now
